mix.exs

```diff
@@ -36,7 +36,7 @@ defmodule MetaLogger.MixProject do
 
       # Middleware dependencies
       {:miss, "~> 0.1", optional: true},
-      {:tesla, "~> 1.4", optional: true}
+      {:tesla, "~> 1.14", optional: true}
```

Comment:
I suggest not forcing the version here, since the library works for `1.4+`. If you agree, you can also remove the `CHANGELOG.md` change.

The change in the `mix.lock` should be kept.